controller/src/command.cpp

```diff
@@ -164,16 +167,15 @@ void command_responseSend(uint8_t cmd, char *packet, uint8_t len) {
 }
 
 static float convIntTofloat(char *data) {
-
     float f;
 
-#ifdef LITTLE_ENDIAN
+#ifndef __FLOAT_WORD_ORDER__
+#error "Your compiler isn't defining the __FLOAT_WORD_ORDER__ macro?"
+#elif __FLOAT_WORD_ORDER__ == __ORDER_LITTLE_ENDIAN__
     /* Convert big endian integer to little endian float */
-    uint32_t data_bEndian = 0;
-    uint32_t data_lEndian = 0;
-
+    uint32_t data_bEndian;
```

Comment:
Is it not good practice to go ahead and zero data_bEndian out as in the lines above? Genuinely a question.

Reply:
I think it's not, because the program would be broken if we ever read the 0.

If we don't initialize it, we have some hope that if we read the variable before it's initialized, we'll get a compiler warning, or (once we get it set up) asan will complain.  Whereas if we initialize it to 0, we're telling all of our tools that 0 is a valid value to read.